packages/components/src/tokens.ts

```diff
@@ -184,5 +184,15 @@ export const tokens = createTokens({
   size,
   space,
   zIndex,
-  radius,
+  radius: {
+    0: 0,
+    4: 4,
+    6: 6,
+    8: 8,
+    10: 10,
+    12: 12,
+    16: 16,
+    20: 20,
+    100: 100,
```

Comment:
Question here is - is 100px ready enough? In Figma it's defined as 50%. We might make it way more (1000?) and maybe name it "max" as an exception? Wdyt?

Reply:
The right word for radiuses is `full` imo. I agree, we could set it to some high value (e.g. 1000px) as React Native does not support percentage values.

Reply:
Sounds good to me